### DISCUSSION TITLE
xcodebuild error

### DISCUSSION THREAD

**Discussion by emeryk1005** [WalletConnect/walletconnect-monorepo#2338] [Developer Support]
When I use the xcodebuild command to compile ipa, I will encounter such an error
error: Resource "/Users/xxx/iOS/wallet-ios/Pods/./build/Release-iphoneos/WalletConnectSwiftV2/WalletConnect_WalletConnectRelay.bundle" not found. Run 'pod install' to update the copy resources script.

Xcode shows that there will be such a folder, as shown in the figure
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/77663523/3f25b931-bf31-47e5-98d2-6fe0185c440e)

I don't know exactly what is going on here and how should I handle this situation. I tried to set SYMROOT or BUILD_DIR in the command, but still this error

**Comment by finessevanes** [comment]
@emeryk1005 can you share the name of the SDK you're using?

**Comment by finessevanes** [comment]
Closing this discussion due to three weeks of inactivity.
